spec/cli/sub_spec.rb

```diff
@@ -7,10 +7,10 @@
 
   describe 'add' do
     context 'with correct arguments' do
-      let(:argv) { %w[sub add https://my-service.dev cats dogs -b bus.dev -t s3cr3t] }
+      let(:argv) { %w[sub add https://my-service.dev cats dogs -b bus.dev -t s3cr3t --batch-size 30] }
```

Comment:
would be great to pass in the latency option as well and have that covered

Reply:
yeah good call. Will do that